animations.md

```diff
@@ -85,7 +106,90 @@ from 1.0 to 0.0). Similarly, you can create a
 whose value is adjusted by a [curve](http://docs.flutter.io/flutter/animation/Curves-class.html).
 
-## Tweens
+### CurvedAnimation
+
+Use the [`CurvedAnimation`](http://docs.flutter.io/flutter/material/CurvedAnimation-class.html)
+class to apply a non-linear curve to an animation object it wraps.
+By applying different curve types, you can visually modify the spatial movement
+pattern for the wrapped animation object.
+
+The
+[`Curves`](http://docs.flutter.io/flutter/animation/Curves-class.html) class
+lists the available curve types, such as
+[`bounceIn`](http://docs.flutter.io/flutter/animation/Curves/bounceIn-constant.html),
+[`easeOut`](http://docs.flutter.io/flutter/animation/Curves/easeOut-constant.html),
+and others.
+
+The following snippet shows how you might create a curved animation object that
+wraps a linear animation produced by an animation controller and applies the
+[`easeIn`](http://docs.flutter.io/flutter/animation/Curves/easeIn-constant.html)
+curve type to it.
+
+```dart
+final AnimationController controller =
+    new AnimationController(duration: const Duration(milliseconds: 500));
+
+final curve = new CurvedAnimation(
+    parent: controller,
+    curve: Curves.easeIn
+);
+```
+
+### Tween
+
+The [`Tween`](http://docs.flutter.io/flutter/material/Tween-class.html) class
+represents a changing value between a range with a
+[`begin`](http://docs.flutter.io/flutter/material/Tween/begin.html) and
+[`end`](http://docs.flutter.io/flutter/material/Tween/end.html) value. To get
+the interpolated value for a given animation, call the
+[`evaluate()`](http://docs.flutter.io/flutter/animation/Tween/evaluate.html)
+method on the Tween object. You can override the
+[`lerp()`](http://docs.flutter.io/flutter/material/Tween/lerp.html)
+(or *linear interpolation*) method for the
+[`Tween`](http://docs.flutter.io/flutter/material/Tween-class.html) object,
+which returns a value based on a given animation clock time.
+
+You can use these subclasses of
+[`Tween`](http://docs.flutter.io/flutter/material/Tween-class.html) to apply
+specialized interpolations to your animations:
+
++ [`ColorTween`](http://docs.flutter.io/flutter/animation/ColorTween-class.html):
+Interpolation between colors.
++ [`SizeTween`](https://docs.flutter.io/flutter/animation/SizeTween-class.html):
+Interpolation between rectangle sizes.
++ [`ThemeDataTween`](https://docs.flutter.io/flutter/material/ThemeDataTween-class.html):
+Interpolation between
+[`ThemeData`](https://docs.flutter.io/flutter/material/ThemeData-class.html) objects.
+
+The following snippet shows how you might create a
+[`Tween`](http://docs.flutter.io/flutter/material/Tween-class.html) object that
+interpolates between the double values of -200.0 to 0.0.
+
+```dart
+final Tween doubleTween = new Tween<double>(begin: -200.0, end: 0.0);
+```
+
+You can also specify a progression between two colors. For example:
+
+```dart
+final Tween colorTween =
```

Comment:
I'd drop the `final` from all these example.  Here I'd use the more specific ColorTween type.